### ISSUE TITLE
no need to introduce extra python env

### ISSUE THREAD

**Issue by calidion** [valdrinkoshi/virtual-scroller#140]
```
$ npm install
$ python -m SimpleHTTPServer 8081
```
can be replaced by :
```
$ npm install
$ npm install -g http-server   // If http-server is not installed.
$ http-server -p 8081
```
no need to introduce extra python env

**Comment by valdrinkoshi** [comment]
Hi @calidion, the intent of that part is to suggest "serve the files however you'd like", and we provided a python script - but it's totally fine to serve in any other way. This is being addressed in #138.

**Comment by valdrinkoshi** [comment]
Fixed in #138

**Comment by calidion** [comment]
Yes. But since you've been using npm in the project already, it is nature to install packages from npm which are already in existence not to introduce a new language and package manager.
